src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -611,15 +612,29 @@ internal int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancella
 
                     if (Arguments.SourceLink != null)
                     {
-                        sourceLinkStreamOpt = OpenFile(Arguments.SourceLink, consoleOutput, FileMode.Open, FileAccess.Read, FileShare.Read);
+                        var sourceLinkStreamOpt = OpenFile(
```

Comment:
It's 160 cols, which means I can't see the whole line on my 15" laptop. I consider that too long.